recipes/BEHST/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+
+./download_behst_data.sh
```

Comment:
Okay, I added the followed commands to `build.sh`:
`mkdir -p $PREFIX/bin`
`cp $SRC_DIR/bin/* $PREFIX/bin`
`cd $PREFIX/bin`

and it worked on Travis. Is this version alright?